packages/cactus-cmd-api-server/src/test/typescript/unit/grpc-proto-gen-ts-client-healthcheck.test.ts

```diff
@@ -9,80 +9,95 @@ import { AuthorizationProtocol } from "../../../main/typescript/public-api";
+
+describe("ApiServer", () => {
+  let apiServer: ApiServer;
+
+  afterAll(async () => {
+    await apiServer.shutdown();
+  });
+
+  test("runs gRPC TS-proto web services", async () => {
+    const configService = new ConfigService();
+    const apiSrvOpts = await configService.newExampleConfig();
+    apiSrvOpts.authorizationProtocol = AuthorizationProtocol.NONE;
+    apiSrvOpts.configFile = "";
+    apiSrvOpts.logLevel = logLevel;
+    apiSrvOpts.apiCorsDomainCsv = "*";
+    apiSrvOpts.apiPort = 0;
+    apiSrvOpts.grpcPort = 0;
+    apiSrvOpts.crpcPort = 0;
+    apiSrvOpts.cockpitPort = 0;
+    apiSrvOpts.grpcMtlsEnabled = false;
+    apiSrvOpts.apiTlsEnabled = false;
+    apiSrvOpts.plugins = [];
+    const config = await configService.newExampleConfigConvict(apiSrvOpts);
+
+    apiServer = new ApiServer({
+      config: config.getProperties(),
+    });
+
+    const startResponse = apiServer.start();
+    expect(startResponse).toResolve();
+
+    const addressInfoGrpc = (await startResponse).addressInfoGrpc;
+    const { address, port } = addressInfoGrpc;
+    const grpcHostAndPort = `${address}:${port}`;
+
+    const apiClient =
+      new default_service.org.hyperledger.cactus.cmd_api_server.DefaultServiceClient(
+        grpcHostAndPort,
+        grpc.credentials.createInsecure(),
+      );
+    expect(apiClient).toBeTruthy();
+
+    const responsePromise =
+      new Promise<health_check_response_pb.org.hyperledger.cactus.cmd_api_server.HealthCheckResponsePB>(
+        (resolve, reject) => {
+          apiClient.GetHealthCheckV1(
+            new empty.google.protobuf.Empty(),
+            (
+              error: grpc.ServiceError | null,
+              response?: health_check_response_pb.org.hyperledger.cactus.cmd_api_server.HealthCheckResponsePB,
+            ) => {
+              if (error) {
+                reject(error);
+              } else if (response) {
+                resolve(response);
+              } else {
+                throw new RuntimeError("No error, nor response received.");
+              }
+            },
+          );
+        },
+      );
+
+    await expect(responsePromise).resolves.toMatchObject({
+      toObject: expect.toBeFunction(),
+    });
+
+    const resPb = await responsePromise;
+    const res = resPb.toObject();
+
+    expect(res).toMatchObject({
+      createdAt: expect.toBeDateString(),
+      memoryUsage: expect.objectContaining({
+        rss: expect.toBeNumber(),
+      }),
+      success: true,
+    });
+    // await t.doesNotReject(responsePromise, "No error in healthcheck OK");
+    // const res = await responsePromise;
+
+    // const resHc = res?.toObject();
+
+    // t.ok(resHc, `healthcheck response truthy OK`);
+    // t.ok(resHc?.createdAt, `resHc.createdAt truthy OK`);
+    // t.ok(resHc?.memoryUsage, `resHc.memoryUsage truthy OK`);
+    // t.ok(resHc?.memoryUsage?.rss, `resHc.memoryUsage.rss truthy OK`);
+    // t.ok(resHc?.success, `resHc.success truthy OK`);
+    // t.end();
```

Comment:
Can this commented code be removed?

Reply:
@outSH Oops, thank you for catching that!